src/entries/popup/pages/send/ToAddressInput.tsx

```diff
@@ -358,7 +361,7 @@ export const ToAddressInput = React.forwardRef<InputRefAPI, ToAddressProps>(
       <>
         <DropdownInputWrapper
           zIndex={2}
-          dropdownHeight={452}
+          dropdownHeight={452 - (testnetMode ? TESTNET_MODE_BAR_HEIGHT : 0)}
```

Comment:
Should we move this 452 into core utils dimensions? Not sure if we're planning to use anywhere else

Reply:
hm i don't think we have the same height anywhere else